src/theme/material/templates/plugin-pager-target.tpl.html

```diff
@@ -0,0 +1,15 @@
+<ng-template key="plugin-pager-target.tpl.html" let-$target>
+<div style="position:fixed; top: 20px; left: 20px"
+	  layout="column"
+	  flex="100">
+	<mat-card>
+		<mat-card-content>
+			<mat-input-container class="q-grid-input-container">
+				<input matInput type="number" #input
```

Comment:
Can you please add min and max attribute to disallow printing of incorrect page values?

Reply:
Yes sure, this task is not yet completed, will work on this soon)